os-eap-legacy/os-eap-s2i/module.yaml

```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: os-eap-s2i
+version: '1.0-kie'
+description: Legacy os-eap-s2i script package.
+modules:
+  install:
+  - name: jboss.container.eap.s2i.bash
```

Comment:
This didn't worked, as we don't have the `jboss.container.eap.s2i.bash` module present. The image build fails with error:
```
cekit.errors.CekitError: Could not locate module jboss.container.eap.s2i.bash version None.  Please verify that it is included in one of the specified module repositories.
```
Instead of `os-eap-s2i` module we should migrate the `jboss.container.eap.s2i.bash` from the legacy branch as all this one is doing is installing that one, I tested the image build and test with migrating that module, that works

Reply:
Yes, thanks, I have copied the correct module.